app/controllers/store_controller.rb

```diff
@@ -149,7 +154,11 @@ def process_donation
     redirect_route = quick_donate_path(:customer_id => @customer.id, :account_code_string => @account_code.code, :donation => @amount)
     @amount > 0 or return redirect_to(redirect_route, :alert => 'Donation amount must be provided')
     # Given valid donation, customer, and charge token, create & place credit card order.
-    @gOrderInProgress = Order.new_from_donation(@amount, @account_code, @customer)
+    @gOrderInProgress = Order.new_from_donation(@amount, Donation.default_code, @customer)
```

Comment:
why is this `Donation.default_code` rather than `@account_code.code` ?

Reply:
i changed it when i was trying to separate the donate_to_specific_fund functionality and recurring_donation functionality into different PRs, but i've added it back now